debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/converters/sink/SinkConverter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.converters.sink;
```

Comment:
```suggestion
package io.debezium.connector.mongodb.sink.converters;
```

I don't think we should mix different converter types into the same namespace.

Reply:
done